packages/playwright-core/bundles/utils/package.json

```diff
@@ -21,13 +22,14 @@
     "progress": "2.0.3",
     "proper-lockfile": "4.1.2",
     "proxy-from-env": "1.1.0",
+    "rimraf": "^3.0.2",
     "socks-proxy-agent": "6.1.1",
     "stack-utils": "2.0.5",
-    "rimraf": "3.0.2",
     "ws": "8.4.2"
   },
   "devDependencies": {
     "@types/debug": "^4.1.7",
+    "@types/glob": "^7.2.0",
```

Comment:
`@types/glob` version does not match `glob` version.

Reply:
Best effort.